tests/playwright/pages/editor-page.js

```diff
@@ -242,4 +277,57 @@ module.exports = class EditorPage extends BasePage {
 	async removeFocus() {
 		await this.getPreviewFrame().locator( '#elementor-add-new-section' ).click( { button: 'right' } );
 	}
+
+	/**
+	 * Hide all editor elements from the screenshots.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async hideEditorElements() {
+		const css = '<style>.elementor-element-overlay,.elementor-empty-view,.elementor-widget-empty,.e-view{opacity: 0;}.elementor-widget,.elementor-widget:hover{box-shadow:none!important;}.elementor-add-section-inner {border: none !important;background-color: #cccccc !important;}</style>',
+			widgetContainer = await this.addElement( { elType: 'container' }, 'document' ),
+			htmlWidget = await this.addWidget( 'html', widgetContainer );
+
+		await this.getPreviewFrame().waitForSelector( '.elementor-widget-html' );
+		await this.selectElement( htmlWidget );
+		await this.page.locator( '.elementor-control-type-code textarea' ).fill( css );
+	}
```

Comment:
@hein-obox I am not sure I like this. Is there not another way to do this? What if you just click on `#elementor-mode-switcher-inner`, it also removes the editor elements (when in Preview mode)?

See: https://github.com/elementor/elementor-pro/blob/release/3.8.0/tests/playwright/sanity/modules/loop-builder/module.test.js#L190-L192

Reply:
@louiswol94 

Implemented.